src/c++/perf_analyzer/genai-perf/genai_perf/wrapper.py

```diff
@@ -37,15 +37,15 @@
 class Profiler:
```

Comment:
Looks great overall, thanks for doing this--did this pass your manual testing?

Reply:
@matthewkotila yessir